packages/php/remote-specs-validation/package.json

```diff
@@ -1,9 +1,15 @@
 {
 	"name": "@woocommerce/remote-specs-validation",
 	"description": "Remote specs testing suite",
+	"version": "1.0.0",
```

Comment:
This should be added to composer.json. In PHP projects composer.json is used by Packagist for to determine version. I'd remove this from here to avoid confusion.

Reply:
Thanks! Added in https://github.com/woocommerce/woocommerce/pull/44484/commits/161b4d2737097ff9973b1d5d49feef4616ffd9fe

Reply:
> I'd remove this from here to avoid confusion.

@moon0326 Would you like to remove the version line from `package.json`? I can agree to reduce confusion since this won't be published to NPM